roles/taca/tasks/main.yml

```diff
@@ -0,0 +1,38 @@
+- name: Fetch TACA-ngi-pipeline from GitHub
+  git: repo={{ taca_ngi_repo }}
+       dest={{ taca_ngi_dest }}
+       version={{ taca_ngi_version }}
+       force=yes
+  tags: taca
+
+- name: Install TACA-ngi-pipeline requirements
+  shell: "{{ ngi_pipeline_venv }}/bin/pip install" -r "{{ taca_ngi_dest }}/requirements.txt"
+  tags: taca
+
+- name: Install TACA-ngi-pipeline
+  shell: cd "{{ taca_ngi_dest }}" && "{{ ngi_pipeline_venv }}/bin/pip" install .
```

Comment:
I've never tried running a `shell` command with mixed quotes like that. I've always quoted the whole string, or nothing. Do you know if this will work?